src/app/tasks/modals/grade-task-modal/grade-task-modal.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { gradeService } from 'src/app/ajs-upgraded-providers';
+
+@Component({
+  selector: 'grade-task-modal',
```

Comment:
Shouldn't the selector start with ' df ` as ` ./eslint.rc ` defined. but the CI does not give any warning and we never use this component as a HTML tag so maybe leave it is fine. What do you think?